static/js/filter.js

```diff
@@ -1,430 +1,176 @@
-            filterCategory.close();
-            
-            const isSearchActive = (filterContainer.classList.contains("is-filtered-by-search"));
-
-            // Reset back to current search query, rather than clearing all filters
-            if (isSearchActive && currentFilteredBySearchAliases && (currentFilteredBySearchAliases.length > 0) ) {
-                
-                currentFilteredBySearchAliases.forEach(alias => {
-                    alias.classList.remove("is-hidden");
-                });
-
-                return;
-            }
-            
-            // Full reset
-            aliases.forEach(alias => {
-                alias.classList.remove("is-hidden");
-            });
-        },
-        apply: (e) => {
-            e.preventDefault();
-
-            const options = [];
-
-            filterCategoryCheckboxes.forEach(checkbox => {
-                if (!checkbox.checked) {
-                    return;
-                }
-
-                options.push(checkbox.dataset.categoryType);
-            });
-
-            const isSearchActive = (filterContainer.classList.contains("is-filtered-by-search"));
-
-            if (!isSearchActive) {
-                filterContainer.classList.add("is-filtered-by-category");
-            }
-
-            filterCategory.filter(options)
-            filterCategory.close();
-        },
-        close: () => {
-            toggleAliasCategoryBar();
-        },
-        oppositeCheck: (e) => {
-            const currentCategory = e.target;
-
-            if (!currentCategory.checked) {
-                return;
-            }
-            
-            const currentParentCategory = currentCategory.dataset.parentCategory;
-            
-            filterCategoryCheckboxes.forEach(checkbox => {
-                if ( (currentCategory !== checkbox) && (checkbox.dataset.parentCategory === currentParentCategory) && checkbox.checked) {
-                    checkbox.checked = !checkbox.checked;
-                }
-            });
+    });
 
-        },
-        open: () => {
-            filterCategoryCheckboxes[0].focus();
-            document.addEventListener("click", filterCategory.categoryMenuOpenListener, false);
-            document.addEventListener("keydown", filterCategory.categoryMenuEscListener, false);
-        },
-        filter: (options) => {
-            if (options.length < 1) {
+    function setCategoryFilters(categories) {
+        activeCategoryFilters = categories;
+        applyFilters();
+    }
+    filterCategoryCheckboxes[0].form.addEventListener("submit", (event) => {
+        event.preventDefault();
+        const categoriesToSet = Array.from(filterCategoryCheckboxes)
+            .filter(checkbox => checkbox.checked)
+            .map(checkbox => checkbox.dataset.categoryType);
+        setCategoryFilters(categoriesToSet);
+        closeCategoryMenu();
+    });
+    filterCategoryCheckboxes[0].form.addEventListener("reset", (event) => {
```

Comment:
nit: unused event arg
```suggestion
    filterCategoryCheckboxes[0].form.addEventListener("reset", () => {
```

Reply:
Fixed in 19bf72a48b16d8c1408f2229478cb62040bbe1dc.